packages/docs/getting-started/running-a-validator.md

```diff
@@ -14,6 +14,19 @@ If you are starting up a validator, please consider leaving it running for a few
 
 ## **Prerequisites**
 
+### Hardware requirements
+
+Because Celo network is based in Proof of Stake, the hardware requirements are not very high. Proof of Stake consensus is not so CPU intensive as Proof of Work but has a higher requirements of network connectivity and lantency. Here you have a list of the standard requirements for running a validator node:
+
+- Memory: 8 GB RAM
+- CPU: Quad core 3GHz (64-bit)
+- Disk: 256 GB of SSD storage
+- Network: 1 GB input/output dual Ethernet
+
```

Comment:
Do we want to put a min. recommended internet connection bandwidth?

Reply:
yep, I would say 1gb is the min. network requirement. I clarified that in the bullet points. Thanks for the suggestion